database.sql

```diff
@@ -175,3 +176,26 @@ INSERT INTO `Student_Reviews` (`student_id`, `testimonial`) VALUES
 (4, 'Je n’ai jamais porté une grande importance aux diplômes, mais plutôt à mes compétences, à ce que je suis capable de faire concrètement. Je cherchais à allier la partie technique et le côté relationnel et fonctionnel dans la suite de ma carrière'),
 (5, 'C’était très intense, j’ai eu une promotion en or et beaucoup d’entraide et ça ça a été très important.'),
 (6, 'Peu importe ce que l’on a fait avant, c’est toujours une force, et non une faiblesse.');
+
+
+--
+-- Contenu de la table `Course`
+--
+
+INSERT INTO `Course` (`id` , `name` , `description` , `capacity` , `location`, `date`, `duration` , `degree` , `financing_supported` , `discipline_id`, `url_image`) VALUES 
```

Comment:
remove id from this sql request (and values associated) You ain't gonna need it since ID is auto generated and auto incremented

Reply:
remove succesfully